### PR TITLE
Add homepage to providers.json

### LINKED ISSUE #6: Add homepage_url to all entries
According to the current develop branch OPTiMaDe (since PR Materials-Consortia/OPTiMaDe#228) all entries MUST have homepage_url. This needs to be fixed in this repository.

### PR DESCRIPTION
**Pull request by CasperWA**
Fixes #6 

Two commits:
- The first adds the `homepage` property for all providers, setting it to `null`.
- The second updates this property for the providers: `aiida`, `exmpl`, `mcloud`, `optimade`.

### CODE REVIEW

File: src/v0.10/providers.json [hunk 1]
```diff
@@ -96,7 +106,8 @@
       "attributes": {
         "name": "open database of xtals",
         "description": "",
-        "base_url": "http://odbx.science/optimade"
+        "base_url": "http://odbx.science/optimade",
+        "homepage": null
```
Comment (ml-evs) [inline src/v0.10/providers.json]:
```suggestion
        "homepage": "http://odbx.science"
```

**Comment by ltalirz** [comment]
Looks good to me, no need to wait for my approval.

**Comment by CasperWA** [comment]
@merkys you probably want to update homepage for the various COD providers :) I wasn't sure what to use, so left it as `null`.

**Comment by merkys** [comment]
> @merkys you probably want to update homepage for the various COD providers :) I wasn't sure what to use, so left it as `null`.

Thanks, opened #9 :)

**Pull request merged**